utils/format.sh

```diff
@@ -13,4 +13,4 @@ fi
 find lib tests/unittests/ tools/ include examples \
      -name \*.h -print0 \
      -o -name \*.cpp -print0 \
-    | xargs -0 -P8 -n1 clang-format -i
+    | xargs -0 -P8 -n1 /usr/bin/clang-format-6.0 -i
```

Comment:
that should not be in this PR (stick to a certain version of clang-format is a good idea though).

Reply:
I don't have clang-format-6.0 on my computer.

Reply:
sorry about this

Reply:
sorry about this